shared/desktop/app/dock-icon.desktop.tsx

```diff
@@ -1,22 +0,0 @@
-import * as SafeElectron from '../../util/safe-electron.desktop'
-
-const changeDock = (show: boolean) => {
```

Comment:
having a standalone file for this tiny function that's only imported from one place hurts more than it helps